funnel/templates/project_comments.html.jinja2

```diff
@@ -46,7 +46,15 @@
               {{ commenttree(comments, project.commentset, project, csrf_form) }}
             </ul>
           {% endif %}
-          {{ commentformtemplate(project, commentform, delcommentform, current_auth) }}
+          {% if project.features.post_comment() %}
+            {{ commentformtemplate(project, commentform, delcommentform, current_auth) }}
+          {% else %}
+            {% if project.features.has_tickets() %}
```

Comment:
We've set a convention. These should be named `comments__new` (double underscore) and `tickets` respectively.